src/hangar/__init__.py

```diff
@@ -4,3 +4,15 @@
 from .repository import Repository
 
 __all__ = ['Repository', 'serve']
+
+try:
+    from .dataloaders import make_tf_dataset
```

Comment:
I would skip the indirection; exposing the same function in multiple different modules isn't particularly pythonic. I'd just pick one place to get the functions from; it doesn't really matter where that place is (`hangar.make_xxx_dataset` or `hangar.dataloaders.make_xxx_dataset` or `hangar.dataloaders.tfloader.make_tf_dataset` all work). I think that I lean towards the top level, but that's a weak preference.